.github/workflows/python.yml

```diff
@@ -216,6 +216,9 @@ jobs:
       - name: Run tests
         run: cd rerun_py/tests && pytest
 
+      - name: Run e2e test
+        run: scripts/run_python_e2e_test.py
```

Comment:
If we are running this as part of the wheel job, there's not a great reason to run:
```
subprocess.Popen(["just", "py-build", "--quiet"], env=build_env).wait()
```
below.  The wheel is installed into the local environment just a few lines up from here.

Reply:
True, but when running it from the command line it's quite useful… I guess I could make it opt-out

Reply:
Let's see how much time it adds to the CI

Reply:
> True, but when running it from the command line it's quite useful… I guess I could make it opt-out

I believe you can silently skip it if the `CI` env var is set.